test/vm.systemd-main

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -eu
+
+grep -q "platform:f*" /etc/os-release
```

Comment:
This is meant to stop non-Fedora triggers early, I suppose?

Reply:
Yes